scripts/sync_milestones.js

```diff
@@ -152,17 +123,21 @@ module.exports = {
   },
 
   // Find the milestone matching the one with the given title.
-  // This matches on 'FxA-N' feature number if present in the title,
+  // This matches on "FxA-NNN" feature number if present in the title,
   // otherwise falls back to a simple string compare.
 
   findMatchingMilestone: function findMatchingMilestone(title, milestones) {
-    var featureMatch = FEATURE_NUMBER_RE.exec(title)
+    if (milestones[title]) {
+      return milestones[title]
+    }
+    var featureRE = /(FxA-\d+):.*/
```

Comment:
This is slightly different than the regex specified at top of file, ` /(FxA-\d+): .*/`, which has a space between colon and period. Is this ok?

Reply:
Nice catch.  I found that when creating the milestones by hand, sometimes people would leave out the space and then this script wouldn't work quite right.  So I think it's OK to treat the space as optional.  Make sense?